clients/lsp-json.el

```diff
@@ -65,24 +65,26 @@
 (defvar lsp-json--major-modes '(json-mode jsonc-mode)
   "List of supported JSON major modes.")
 
-(defvar lsp-json--schema-associations
-  `(:/*.css-data.json ["https://raw.githubusercontent.com/Microsoft/vscode-css-languageservice/master/docs/customData.schema.json"]
-    :/package.json ["http://json.schemastore.org/package"]
-    :/*.html-data.json ["https://raw.githubusercontent.com/Microsoft/vscode-html-languageservice/master/docs/customData.schema.json"]
-    :/*.schema.json ["http://json-schema.org/draft-07/schema#"]
-    :/bower.json ["http://json.schemastore.org/bower"]
-    :/composer.json ["http://json.schemastore.org/composer"]
-    :/tsconfig.json ["http://json.schemastore.org/tsconfig"]
-    :/tsconfig.*.json ["http://json.schemastore.org/tsconfig"]
-    :/typings.json ["http://json.schemastore.org/typings"]
-    :/.bowerrc ["http://json.schemastore.org/bowerrc"]
-    :/.babelrc ["http://json.schemastore.org/babelrc"]
-    :/.babelrc.json ["http://json.schemastore.org/babelrc"]
-    :/babel.config.json ["http://json.schemastore.org/babelrc"]
-    :/jsconfig.json ["http://json.schemastore.org/jsconfig"]
-    :/jsconfig.*.json ["http://json.schemastore.org/jsconfig"]
-    :/project.json ["http://json.schemastore.org/project"]
-    :/omnisharp.json ["http://json.schemastore.org/omnisharp"])
+(defvar lsp-json-schema-associations
```

Comment:
Did the `plist` looks odd for this?
Also, the `lsp-json--schema-associations` supposed to be private. There's `lsp-json-schemas` for customizing.

Reply:
Since the contents of `hashtable` cannot be easily viewed from `describe-symbol`, I put this as `plist` here though.

Reply:
It is subjective I guess, I can revert this one if you insist.

Reply:
I do prefer `plist` for easy viewing from `describe-symbol`.

Reply:
fixed in the latest version.